ynr/apps/candidates/models/popolo_extra.py

```diff
@@ -511,6 +511,19 @@ def get_absolute_queued_image_review_url(self):
         url = reverse("photo-review-list")
         return f"{url}?{querystring}"
 
+    @property
+    def is_welsh_run(self):
```

Comment:
This change/commit shouldn't be appearing as it is already included in the base branch. I think you need to ensure your local `feature/previous-political-affiliation` branch is up to date, then run `git rebase feature/previous-political-affiliation` and then push this branch with `force-with-lease`